src/data/set/pointwise/smul.lean

```diff
@@ -378,6 +378,15 @@ end vsub
 
 open_locale pointwise
 
+@[to_additive] lemma image_smul_comm [has_smul α β] [has_smul α γ] (f : β → γ) (s : set β) (a : α) :
+  (∀ b, f (a • b) = a • f b) → f '' (a • s) = a • f '' s :=
+image_comm
+
+@[to_additive] lemma image_smul_distrib [monoid α] [monoid β] [monoid_hom_class F α β] (f : F)
```

Comment:
Could consider a TODO comment about generalizing to `mul_distrib_mul_action`, but not necessary.

Reply:
Oh: does this work for `has_mul` instead of `monoid`?

Reply:
Oh yes indeed